friends/src/App.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react';
+import { Route, Link } from 'react-router-dom';
+import './App.css';
+import axios from 'axios';
+
+import FriendsList from './components/FriendsList';
+import SubmitNewFriend from './components/SubmitNewFriend';
+import EditFriend from './components/EditFriend';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      name: "",
+      age: null,
+      email: ""
+    }
+  }
+
+  componentDidMount() {
+    axios
+        .get("http://localhost:5000/friends")
+        .then(response => {
+            this.setState({friends: response.data})
+        })
+        .catch(err => {
+            console.log(err);
+        })
+  }
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value,
+    })
+  }
+
+  addNewFriend = event => {
+    const newFriendObj = {
+      name: this.state.name,
+      age: this.state.age,
+      email: this.state.email
```

Comment:
This object could be a little cleaner with a bit of [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring):
```javascript
addNewFriend = event => {
  // makes variables for name, age, and email w/ values off of state
  const { name, age, email } = this.state;
  // assigns these keys with their variables' corresponding values (shorthand)
  const newFriendObj = { name, age, email };
  // ...
};
```